src/routes/institution-route.js

```diff
@@ -0,0 +1,12 @@
+import express from 'express'
+import * as controller from '../controllers/institution-controller.js'
+
+const institutionRouter = express.Router({ mergeParams: true })
+
+institutionRouter.get('/', controller.fetchProjectInstitutions)
+institutionRouter.get('/search', controller.searchInstitutions)
+
+institutionRouter.post('/create', controller.addInstitutionToProject)
```

Comment:
I like that it's combined since most of the code is similar.

Now let's make their name similar, change `/create` to `/add`